packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -39,6 +39,11 @@ export interface IButtonProps extends React.HTMLProps<HTMLButtonElement | HTMLAn
   ariaDescription?: string;
 
   /**
+  * Text to render button label. If text is supplied, it will override any string in button children. Other children components will be passed through after the text.
+  */
+  text?: string;
```

Comment:
correct. we'd thought that 'label' was a typical button tag for some reason. Still not sure why other people were using it in their product.